dll/cpl/hotplug/lang/en-US.rc

```diff
@@ -36,4 +48,6 @@ STRINGTABLE
 BEGIN
     IDS_CPLNAME "Safely Remove Hardware"
     IDS_CPLDESCRIPTION "Safely unplug or eject devices from your computer."
+    IDS_UNKNOWN_DEVICE "Unknown Device"
+    IDS_EJECT_ERROR_FORMAT "Failed to remove device (0x%x)\0"
```

Comment:
You will have to duplicate all those strings and controls into all other .rcs. This is part of the implementors job when new stuff is added.

Reply:
I planned to do that after the code review (see TODO list).

Reply:
I see. I will give my approval after they will have been added then.

Reply:
OK.